styles.css

```diff
@@ -428,15 +450,134 @@ span .item3 {
   flex-shrink: 0;
 }
 
-.grdicontainer {
-  display: grid;
-  display: inline-block;
+.work {
+  flex-direction: row;
+  display: flex;
+  padding: 16px 26px 16px 12px;
+  align-items: center;
+  gap: 12px;
+  width: 280px;
+  height: 40px;
+}
+
+.work b {
+  display: flex;
+  width: 237px;
+  flex-direction: columnr;
+  justify-content: start;
+  color: #000;
+  font-family: Poppins, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  flex-shrink: 0;
+  background-color: #fff;
+}
+
+.self {
+  background-color: #6070ff;
+  border-radius: 70px 0 0 0;
+  height: 800px;
+}
```

Comment:
- [ ] Please ensure that your design matches the given mobile design as part of the requirement that says: `You must stick to the design as much as possible (e.g., font, colors, images, text, margins)` using this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&mode=design&t=bI4j1EGgLLXpezhr-0) matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Figma</td>
    </tr>
    <tr>
      <td>

 
![Student View](https://github.com/okayboy12/Portfolio-setup-and-mobile-first/assets/43528594/d067be8a-2165-4023-b11a-b3dc63c85f71)
 
</td>
      <td>

![Figma btn layout](https://github.com/okayboy12/Portfolio-setup-and-mobile-first/assets/43528594/308b420b-b9a5-44c5-94fa-ccfd89b5b78d)

</td>
    </tr>
  </table>